src/webviews/apps/home/home.html

```diff
@@ -204,6 +204,9 @@ <h1 class="alert__title">Get Started with GitLens</h1>
 					</gl-tooltip>
 				</a>
 			</div>
+			<nav class="nav-list">
+				<gl-home-app></gl-home-app>
```

Comment:
Maybe something like:

```suggestion
				<gl-onboarding><span slot="title">Get Started with GitLens</span></gl-onboarding>
```

Reply:
ok